### ISSUE TITLE
why Not work Chat?

### ISSUE THREAD

**Issue by AmSoMad** [microsoft/vscode-copilot-release#118]
- VS Version:  Microsoft Visual Studio Enterprise 2022 (64-bit) v17.6.1
- OS Version: windows10
- Logs: 

[Conversations Information] Activated Conversations version 0.1.651-beta+8606258f14.
[Conversations Information] [ServiceTarget] Conversations service is running. PID: 29140
[Conversations Information] [CopilotAuthTokenManager] Need to get an auth token.
[Conversations Information] [CopilotAuthTokenManager] Status Code: 403, Reason Phrase: Forbidden
[Conversations Information] [CopilotAuthTokenManager] Response Content: HasToken: False, ChatEnabled: , ExpiresAt: 0, RefreshIn: 0, SKU: 
[Conversations Error] [CopilotAuthTokenManager] Response didn't contain a token.
[Conversations Error] [Conversations] Exception in VerifyUserHasChatAccessAsync: System.UnauthorizedAccessException: User is not signed into GitHub and/or doesn't have Copilot chat access.
[ConversationsServiceManager Error] Exception during service initialization: StreamJsonRpc.RemoteInvocationException: User is not signed into GitHub and/or doesn't have Copilot chat access.
   위치: StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__151`1.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: Microsoft.VisualStudio.Conversations.ConversationsServiceManager.<InitializePlannableFunctionsAsync>d__13.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: Microsoft.VisualStudio.Conversations.ConversationsServiceManager.<InitializeAsync>d__10.MoveNext()
RPC server exception:
System.UnauthorizedAccessException: User is not signed into GitHub and/or doesn't have Copilot chat access.
      at Microsoft.VisualStudio.Conversations.Service.ServiceTarget.VerifyUserHasChatAccessAsync(CancellationToken cancellationToken) in D:\a\_work\1\s\Conversations.Service.Implementation\ServiceTarget.cs:line 177
      at Microsoft.VisualStudio.Conversations.Service.ServiceTarget.GetBackendAsync(CancellationToken cancellationToken) in D:\a\_work\1\s\Conversations.Service.Implementation\ServiceTarget.cs:line 157
      at Microsoft.VisualStudio.Conversations.Service.ServiceTarget.InitializePlannableFunctionsAsync(PlannableFunctionsContext context, CancellationToken cancellationToken) in D:\a\_work\1\s\Conversations.Service.Implementation\ServiceTarget.cs:line 217

[Conversations Information] [CopilotAuthTokenManager] Need to get an auth token.
[Conversations Information] [CopilotAuthTokenManager] Status Code: 403, Reason Phrase: Forbidden
[Conversations Information] [CopilotAuthTokenManager] Response Content: HasToken: False, ChatEnabled: , ExpiresAt: 0, RefreshIn: 0, SKU: 
[Conversations Error] [CopilotAuthTokenManager] Response didn't contain a token.
[Conversations Information] info bar closed

--------------------------------------------------

I am a user using the existing copilot.
I'm trying to experience this preview, 
but there seems to be a problem with the authentication part.

GitHub Copilot 1.84.0.1 (v1.84.0.1@760da3695)
[DEBUG] [agent] [2023-05-25T04:46:52.790Z] Agent service starting
[DEBUG] [agent] [2023-05-25T04:46:53.063Z] Telemetry initialized
GitHub Copilot Agent v1.84.0

Copilot Chat -  authentication part
""
You're in!
The GitHub Copilot chat private beta is already enabled on your account!
""

**Comment by shanirinacohen** [comment]
Hi, i got the exact same issue. Can we get help pls? I got the approval on June 7th.
@AmSoMad Was it solved for you? If yes, can you tell me how? Thanks.

**Comment by GaTechThomas** [comment]
I'm finding that on my second computer it's not recognizing my user that has copilot access (which is not the primary user in my Visual Studio).

**Comment by shanirinacohen** [comment]
Not sure i understand, so you're saying you have the same issue on your second computer?

**Comment by AmSoMad** [comment]
hi, @shanirinacohen 

I solved it. However, there were also mistakes on my part.

I am using 2 of them.
Visual Studio, VS Code

I set up different github accounts.
Visual Studio - Accounts that are *unlicensed* to preview,
VS Code - Accounts with Preview Permission
github desktop - account with preview permission

Install the Visual Studio extension,
VS Code also installed the preview.

403 error occurred in both Visual Studio and VS Code.

Log out of all previously logged in information
It worked fine when I logged in only with an account that was allowed to preview.
